src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -117,7 +117,12 @@
    (let [{:keys [id method public-key to symbol amount-text on-result on-error
                  send-transaction-message?]}
          (get-in db [:wallet :send-transaction])
-         db' (assoc-in db [:wallet :send-transaction :in-progress?] false)]
+         db' (assoc-in db [:wallet :send-transaction :in-progress?] false)
+         ; NOTE: we check last 3 screens to understand if wallet modal screen was used
+         ; (not just last one, we need to account for 'Enter PIN' and 'Keycard connect' screens)
+         screens-before (set (take 3 (:navigation-stack db)))
```

Comment:
maybe we could use some temporary value and do not rely on `(take 3 (:navigation-stack db))` doesn't look reliably

Reply:
@flexsurfer I can add flag `:modal?` to `:navigation/screen-params` and set/unset it every time wallet send tx flow is initialized. Is that what you had in mind?

Reply:
yeah smth like that

Reply:
@flexsurfer done, please check